io.catenax.shared.quantity/1.0.1/Quantity.ttl

```diff
@@ -0,0 +1,291 @@
+#######################################################################

```

Comment:
I have created quantity 2.0.0 with samm 2.1.0 version. Would be better if you can enhance the version from that or simple change the example value in quantity 2.0.0 version.

Reply:
@agg3fe first didn't notice. Merged and needed to remove the bot commit (therefore some force pushs). Should now be corrected. Would just go with 2.0.0